tests/version-macros/BUILD.bazel

```diff
@@ -32,6 +33,40 @@ haskell_test(
     ],
 )
 
+haskell_test(
+    name = "version_macros_haskell_module",
+    size = "small",
+    modules = [
+        ":HsLib",
+        ":HscLib",
+        ":Main",
+    ],
+    version = "3.4.5.6",
```

Comment:
The tests only seem to check if the define for `versioned_lib` exist, not the one for `version_macros_haskell_module`. This also seems to be the case in the previously existing test, there's no check for for whether `VERSION_version_macros` is defined. Perhaps we should add checks for those?

Am I misunderstanding something?

Reply:
> not the one for version_macros_haskell_module

I just tried it. `VERSION_version_macros` isn't defined when building `version_macros`. If Cabal does define it, it could make sense to create a feature request.